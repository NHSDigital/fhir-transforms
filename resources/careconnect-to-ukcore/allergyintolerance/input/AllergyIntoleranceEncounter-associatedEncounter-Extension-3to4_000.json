{
  "resourceType": "AllergyIntolerance",
  "extension": [
    {
      "url": "http://hl7.org/fhir/StructureDefinition/encounter-associatedEncounter",
      "valueReference": {
        "reference": "Encounter/7D179839-BECC-49FB-B58D-97627930D360"
      }
    }
  ]
}
