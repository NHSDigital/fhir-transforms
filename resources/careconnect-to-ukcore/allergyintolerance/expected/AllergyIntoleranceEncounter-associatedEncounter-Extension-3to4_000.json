{
  "resourceType": "AllergyIntolerance",
  "encounter": {
    "reference": "Encounter/7D179839-BECC-49FB-B58D-97627930D360"
  }
}